packages/overlay/src/integrations/sentry/data/sentryDataCache.ts

```diff
@@ -241,11 +241,13 @@ function reverseStackTraces(errorEvent: SentryErrorEvent): void {
   if (!errorEvent.exception || !errorEvent.exception.values) {
     return;
   }
+
+  // TODO: To check if stacktrace is already reverse or not for php and python.
```

Comment:
We have a sample php error (`envelope_php_error.txt`) in the overlay package. It should be sent to the overlay if you run `pnpm sample` in the overlay package. Let's check if we can remove the commented code before merging this.

Reply:
yes @Lms24 checked that. And based on @HazAT response here - https://discord.com/channels/621778831602221064/1195676075271913502/1195677537683120139

I made this code comment. in case if we wanna check and come back to this later.

let me know if we should remove the commented code.

Reply:
Ah sorry, I missed in your reply that you checked the sample data. 

Yes, I'd prefer if we removed the commented code. We can always revert this PR or go back in the git history if we encounter problems.

Reply:
Yes. 
Done. Removed the comments.